openupgrade_scripts/scripts/purchase_requisition/16.0.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,41 @@
+---Models in module 'purchase_requisition'---
+new model purchase.order.group
+new model purchase.requisition.alternative.warning [transient]
+new model purchase.requisition.create.alternative [transient]
+# NOTHING TO DO
+
+---Fields in module 'purchase_requisition'---
+purchase_requisition / product.template         / purchase_requisition (selection): DEL selection_keys: ['rfq', 'tenders']
```

Comment:
@SonCrits I found 1 commit https://github.com/odoo/odoo/commit/00689e7fc61ac08a2f75d856dc077a9ef46b2b26 to remove Call to Tender form. I think you need to migrate old data to Blanket Order. Maybe I didn't understand correctly, please check again